10_Examples/linh-onboarding/include/linh-onboarding/subscriber.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <memory>
+#include "std_msgs/msg/string.hpp"
+#include "std_msgs/msg/float32.hpp"
+#include "rclcpp/rclcpp.hpp"
+
+using std::placeholders::_1;
+
+namespace onboarding{
+    class Subscriber : public rclcpp::Node {
+    public:
+    Subscriber();
+    void topic_callback(const std_msgs::msg::Float32::SharedPtr msg);
+    int add(int a, int b);
+    int subtract(int a, int b);
```

Comment:
If you are making a library for someone else to use, this is where you would add comments to describe how to use the methods.

Probably not necessary for stuff like add(int a, int b), but good for something like superFancyFunction(int a, int q, ...)